### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by harshad16** [thoth-station/workflow-helpers#371]
**Describe the bug**
```
Traceback (most recent call last):
  File "update_keb_installation.py", line 142, in <module>
    update_keb_installation()
  File "update_keb_installation.py", line 58, in update_keb_installation
    project = service.get_project(namespace=_SLUG.split("/")[0], repo=_SLUG.split("/")[1])
IndexError: list index out of range
```

**Comment by KPostOffice** [comment]
I think I found the issue, but I'm not sure the best way to go about solving this.

https://github.com/thoth-station/common/blob/4b948c3a9833ed12897daeedad7e438820fbf733/thoth/common/openshift.py#L300

In common if you pass `foo=None` as part of the parameters it will be changed to `foo=""` instead of the default template parameter. This behavior seems wrong but I'm not sure if it is relied on somewhere.

@fridex

**Comment by KPostOffice** [comment]
Will remove parameter if it is `None` in the calling function as this might be relied on

**Comment by fridex** [comment]
This has been fix by Kevin in thoth-station/common#1212

/close
